src/components/Shared/FieldSet.js

```diff
@@ -1,42 +1,33 @@
-// ** MUI Imports
-import {  Grid, Box, Typography } from '@mui/material'
-
-
-import { useSettings } from 'src/@core/hooks/useSettings'
-
-// ** Resources
-import { TrxType } from 'src/resources/AccessLevels'
-
-const FieldSet = ({
-  children,
- title,
-  ...props
-}) => {
-
+import { Grid, Typography } from '@mui/material'
 
+const FieldSet = ({ children, title, ...props }) => {
   return (
-
-
-
-       <Grid container  spacing={2} sx={{
-            border: '2px solid #0111', 
-            padding: '20px', 
-            position: 'relative',
-            marginY:'10px',
-            marginLeft:'0px',
-            ...props.sx
-          }} >
-            <Typography variant="h6" style={{
-              position: 'absolute',
-              fontSize:'14px',
-              top: '-13px', 
-              backgroundColor: '#F4F5FA', 
-              padding: '0 10px', 
-            }}>{title}</Typography>
-                   {children}
-
-
-        </Grid>
+    <Grid
+      container
+      spacing={2}
+      sx={{
+        border: '2px solid #0111',
+        padding: '10px',
+        position: 'relative',
+        marginY: '10px',
+        marginLeft: '0px',
+        ...props.sx
+      }}
+    >
+      <Typography
+        variant='h6'
+        style={{
+          position: 'absolute',
+          fontSize: '14px',
+          top: '-13px',
+          backgroundColor: '#F4F5FA',
+          padding: '0 10px'
+        }}
+      >
+        {title}
+      </Typography>
+      {children}
+    </Grid>
   )
 }
 
```

Comment:
![image](https://github.com/user-attachments/assets/2dd1527b-88e1-459f-9427-e2660af36828)
there is duplicate of all the values , just check if duplicate is allowed please

Reply:
i will check with Nora